src/pages/partners/affiliate-ib/_dmt5-cards.tsx

```diff
@@ -286,13 +286,25 @@ const StyledButton = styled(Button)`
 const DMT5Cards = ({ data }: DMT5Props) => {
     const dataImages = useStaticQuery(query)
     const [is_calculated, setCalculated] = React.useState(false)
+    const headerRef = useRef(null)
 
     const toggleCalculated = () => {
         setCalculated(!is_calculated)
     }
 
+    const scrollToHeader = () => {
```

Comment:
const useScrollToElement = (targetElement: string, offset = 0) => {
    const clickToScrollHandler = () => {
        const element = document.getElementById(targetElement)
        if (element) {
            window.scrollTo({
                top: element.offsetTop + offset,
                left: 0,
                behavior: 'smooth',
            })
        }
    }
    return clickToScrollHandler
}

export default useScrollToElement
can you try to use this hook instead of ref ?

Reply:
Ok, thanks will have a look